src/test/java/com/marklogic/kafka/connect/sink/HandleWriteFailureTest.java

```diff
@@ -24,7 +24,7 @@ void handleWriteFailure() {
         final String TEST_COLLECTION = "handle-write-failure-test";
 
         WriteBatcherSinkTask task = (WriteBatcherSinkTask) startSinkTask(
-            MarkLogicSinkConfig.DOCUMENT_FORMAT, "json",
+            MarkLogicSinkConfig.DOCUMENT_FORMAT, "JSON",
```

Comment:
Is this implying that "json" is no longer valid? That seems like it would be a breaking change. Lowercase versions of these values should still be accepted.